ddtrace/tracer/textmap.go

```diff
@@ -192,6 +194,14 @@ func (i *withOptionsInjector) injectTextMapWithOptions(spanCtx ddtrace.SpanConte
 		if sp, ok := ctx.samplingPriority(); ok {
 			samplingPriority = sp
 		}
+
+		if e, ok := ctx.meta(ext.Environment); ok {
+			env = e
+		}
+
```

Comment:
Also this one please. Please see https://github.com/DataDog/dd-trace-go/wiki/Style-guidelines#general-guidelines

Reply:
I deleted a bunch more blank lines.